cmd/commands/chaincode/events_test.go

```diff
@@ -138,7 +138,7 @@ var _ = Describe("ChaincodeEventsImplementation", func() {
 			}()
 
 			// give the test case a chance to read from the channel
-			time.Sleep(1 * time.Millisecond)
+			time.Sleep(100 * time.Millisecond)
```

Comment:
It might be easier to move the channel close to a `AfterEach` block instead of sleeping.